tests/foreman/ui/test_subscription.py

```diff
@@ -37,6 +40,14 @@
 pytestmark = [pytest.mark.run_in_one_thread, pytest.mark.skip_if_not_set('fake_manifest')]
 
 
+@pytest.fixture
+def import_future_dated_subscription_manifest(target_sat, function_org):
```

Comment:
How about moving this fixture to taxonomy setup fixtures, so it can be reused in future for new tests ?
```suggestion
def func_future_dated_subscription_manifest(target_sat, function_org):
```

Reply:
Good idea, as of now none other than this test uses `func_future_dated_subscription_manifest`, hence I keep it in `ui/test_subscription.py` 
let me know your thoughts on this.

Reply:
Yes, I know, but lets move it in global taxonomy fixtures along with other Manifester fixtures, so any new tests might use that

Reply:
Can we just keep it in same test module? We could move this into `robottelo/pytest_fixtures/component/subscription.py` while writing new test.

Reply:
Done